ui/imports/utils/qmldir

```diff
@@ -1,5 +1,6 @@
 Audio 1.0 Audio.qml
 Tracer 1.0 Tracer.qml
+DidYouKnowMessages 1.0 DidYouKnowMessages.qml
```

Comment:
I feel like this doesn't belong to Utils?

Reply:
I share the feeling. I've placed it here because it serves more or less the same purpose as Constants. Do you have another place in mind?

Reply:
What about `ui/imports/shared/panels/private`? It's only used by `DidYouKnowSplashScreen.qml`

Reply:
sounds good to me! Yes, it's used only by this component